lib/classes/TradeOffer.js

```diff
@@ -248,19 +255,22 @@ TradeOffer.prototype.removeTheirItems = function(items) {
 
 function addItem(details, offer, list) {
 	if (offer.id) {
-		throw new Error("Cannot add items to an already-sent offer");
+		throw new Error('Cannot add items to an already-sent offer');
 	}
 
-	if (typeof details.appid === 'undefined' || typeof details.contextid === 'undefined' || (typeof details.assetid === 'undefined' && typeof details.id === 'undefined')) {
-		throw new Error("Missing appid, contextid, or assetid parameter");
+	if (typeof details.appid === 'undefined'
+		|| typeof details.contextid === 'undefined'
+		|| (typeof details.assetid === 'undefined'
+			&& typeof details.id === 'undefined')) {
```

Comment:
Line length is one of those things that I prefer to just leave up to "reasonable". As in, "try to keep line lengths reasonable, for the most part, but if some lines need to be longer then that's okay". For that reason, I prefer to leave it out of automated checking.